src/dataLoaders/components/collectorsWizard/verify/VerifyCollectorsStep.tsx

```diff
@@ -84,4 +97,6 @@ const mstp = ({
   token,
 })
 
-export default connect<StateProps, {}, OwnProps>(mstp)(VerifyCollectorStep)
+const connector = connect<StateProps, {}, OwnProps>(mstp)
```

Comment:
I think this only needs `StateProps`, but I'm not sure.

does this work:
```ts
const connector = connect<StateProps>(mstp)

Reply:
It seems to work, @TCL735 what do you think? the `OwnProps` has `CollectorStepProps` which seems to be relevant to the modal.

Reply:
If it works, then i would say this is just an optional TypeScript thing to add explicit types.

https://react-redux.js.org/using-react-redux/usage-with-typescript#typing-the-connect-higher-order-component